benefit/data/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -40,11 +40,36 @@
   {"LiveFrom": "01/01/2018", "ID": "name", "ListElementCode": "lastName", "FieldType": "Text", "ElementLabel": "Last Name", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "role", "ListElementCode": "name", "FieldType": "FixedList", "FieldTypeParameter": "FL_roles", "ElementLabel": "Role", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "ID": "role", "ListElementCode": "description", "FieldType": "Text", "ElementLabel": "Description", "FieldShowCondition": "name=\"Other\"" ,"SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/01/2018", "ID": "reasonableAdjustmentDetails", "ListElementCode": "wantsReasonableAdjustment", "FieldType": "YesOrNo", "ElementLabel": "Wants Reasonable Adjustment", "SecurityClassification": "PUBLIC"},
```

Comment:
Are these going into production straight away, I forgot what needs to be in non prod sheets and what doesn't so approved ether way but might be a good check!

Reply:
Yes they are